ValidationLibrary.Rules.Tests/HasLicenseRuleTests.cs

```diff
@@ -8,43 +8,30 @@
 
```

Comment:
Should be licence everywhere, not license

Reply:
This is a difference between american & british english and GitHub uses "License", so we are going with "License".

Reply:
I tried 😂

Reply:
Understandable. :D